common/internal_model/src/main/scala/weco/catalogue/internal_model/index/WorksIndexConfig.scala

```diff
@@ -37,10 +37,22 @@ object WorksIndexConfig extends IndexConfigFields {
   val merged = WorksIndexConfig(
     Seq(
       keywordField("type"),
+      //
+      // We copy the relationPath field to enable two types of query in the
+      // relation embedder:
```

Comment:
This is also in the API.

Reply:
This is a comment on the config for the merged-works index – there's a separate comment for the API further down.